constants/error-constants.js

```diff
@@ -34,9 +33,10 @@ const ERROR = {
     // List Submissions
     LIST_SUBMISSION_INVALID_STATUS_FILTER: "The status filter is invalid",
     DUPLICATE_STUDY_ABBREVIATION: "Study abbreviation must be a unique value as it already exists in the database.",
-
+    INVALID_SUBMISSION_PERMISSION: "You do not have permission to list submission",
```

Comment:
Can we update this message to "You do not have the correct permissions to list submissions"

Reply:
Thank you Austin. Updated.